website/versioned_docs/version-0.72/native-api/QuirkSettings-api-windows.md

```diff
@@ -0,0 +1,79 @@
+---
+id: version-0.72-QuirkSettings
+title: QuirkSettings
+original_id: QuirkSettings
+---
+
+Kind: `class`
+
+
+
+> **EXPERIMENTAL**
+
+This can be used to add settings that allow react-native-windows behavior to be maintained across version updates to facilitate upgrades. Settings in this class are likely to be removed in future releases, so apps should try to update their code to not rely on these settings.
+
+
+
+## Methods
+### SetAcceptSelfSigned
+`static` void **`SetAcceptSelfSigned`**([`ReactInstanceSettings`](ReactInstanceSettings) settings, bool value)
+
+> **EXPERIMENTAL**
+
+Runtime setting allowing Networking (HTTP, WebSocket) connections to skip certificate validation.
+
+
+
+### SetBackHandlerKind
+`static` void **`SetBackHandlerKind`**([`ReactInstanceSettings`](ReactInstanceSettings) settings, [`BackNavigationHandlerKind`](BackNavigationHandlerKind) kind)
+
+> **EXPERIMENTAL**
+
+By default `react-native-windows` will handle various back events and forward them to JavaScript. Setting this to [`BackNavigationHandlerKind.Native`](BackNavigationHandlerKind#native) prevents `react-native-windows` from handling these events, including forwarding to JavaScript.  This will allow applications to handle back navigation in native code, but will prevent the `BackHandler` native module from receiving events.
```

Comment:
```suggestion
By default `react-native-windows` will handle various back events and forward them to JavaScript. Setting this to [`BackNavigationHandlerKind.Native`](BackNavigationHandlerKind) prevents `react-native-windows` from handling these events, including forwarding to JavaScript.  This will allow applications to handle back navigation in native code, but will prevent the `BackHandler` native module from receiving events.
```